node/src/components/small_network/message.rs

```diff
@@ -432,16 +434,22 @@ mod tests {
 
     /// Serialize a message using the standard serialization method for handshakes.
     fn serialize_message<M: Serialize>(msg: &M) -> Vec<u8> {
-        // The actual serialization/deserialization code can be found at
-        // https://github.com/carllerche/tokio-serde/blob/f3c3d69ce049437973468118c9d01b46e0b1ade5/src/lib.rs#L426-L450
+        let mut serializer = MessagePackFormat;
```

Comment:
Maybe it's worth leaving a comment here why we still have `MessagePack` for the handshake while use `bincode` for everything else.

Reply:
This is the test module. Also, it says "standard serialization method for handshakes" right at the top!

Reply:
OK, I would still be curious why we use different format for handshakes.